src/builder/modules/Generator.res

```diff
@@ -122,10 +122,31 @@ let usePage = id => {
   model := Builder.usePage(model.contents, id)
 }
 
-let fillBackgroundColor = (fillStyle: string) => {
-  model := Generator_Builder.fillBackgroundColor(model.contents, fillStyle)
+let fillBackgroundColor = (~color: string="#ffffff", ()) => {
+  model := Generator_Builder.fillBackgroundColor(model.contents, ~color)
+}
+let drawLine = (
+  dest: Builder.rectangle,
+  ~color: string="#000000",
+  ~width: float=1.0,
+  ~pattern: array<int>=[],
+  ~offset: int=0,
+  (),
+) => {
+  model := Generator_Builder.drawLine(model.contents, dest, ~color, ~width, ~pattern, ~offset)
 }
 
+let drawFold = (dest: Builder.rectangle) => {
```

Comment:
This is great - could we split this into two standard functions?

E.g.

```
let drawMountainFold = (from: Builder.position, from: Builder.position) => ...
let drawValleyFold = (from: Builder.position, from: Builder.position) => ...
```

I like how we can standardize how all folds are drawn in all generators.

Reply:
Mountain fold patterns aren't really used often in papercraft designs- I just used it for the readme as an example of a possible dash pattern (though most papercraft folds do end up being mountain folds in practice). However, if it or some other kind of fold were to become a standard for something, like for cutting, inserting or valley folding, then the parameters in drawFold() could be used to facilitate them without adding resistance to the adoption of the standard.

Reply:
Cool, that makes sense. Just `drawFoldLine()` sounds fine then.